src/components/Cart.vue

```diff
@@ -26,32 +27,67 @@
 </template>
 
 <script>
-import MdButton from './Materialize/mdButton'
-import ProductItem from './ProductInCartItem'
-import { mapState } from 'vuex'
+  import MdButton from './Materialize/mdButton'
+  import ProductItem from './ProductInCartItem'
+  import { mapState } from 'vuex'
 
-export default {
-  components: {
-    ProductItem,
-    MdButton
-  },
-  computed: {
-    ...mapState({
-      cartList: state => state.cart.cartContent,
-      productList: state => state.products
-    })
+  export default {
+    components: {
+      ProductItem,
+      MdButton
+    },
+    data () {
+      return {
+        paypalInstance: null,
+        basketPriceSum: 15.5
+      }
+    },
+    computed: {
+      ...mapState({
+        cartList: state => state.cart.cartContent,
+        productList: state => state.products
+      })
+    },
+    methods: {
+      checkout () {
+        this.paypalInstance.tokenizeAsync({
+          flow: 'checkout',
+          amount: this.basketPriceSum,
+          currency: 'EUR',
+          locale: 'fr_FR'
+        })
+          .then(payload => {
+            // Tokenization succeeded!
+            return window.kuzzle.queryPromise({
+              controller: 'kuzzle-plugin-braintree/braintree',
+              action: 'checkout'
+            }, { body: { payment_method_nonce: payload.nonce } })
+              .then((res) => {
+                // TODO : Handle transaction success
+                // Doc will be exposed soon
+                console.log('Success transaction', res.transaction)
+              })
+          })
+          .catch(err => {
+            // TODO : Handle errors
```

Comment:
Could you put a link of the Braintree error reference in the TODO comment, please? (so that we can leave the TODO here but will know where to look when we'll implement a better error handling)